shared/common-adapters/text.css

```diff
@@ -0,0 +1,295 @@
+/* Used by text on desktop to speed up perf */
```

Comment:
I found a text type that's missing here (`BodyTinyBold`). Do we have a way to make sure this doesn't diverge with `text.meta.*`?

Reply:
longer term we could generate this , but i'm keeping this simple for now

Reply:
added it back in